### PR TITLE
Jtso/list bir

### PR DESCRIPTION
**Pull request by jtsodapop**
Project.bulk_import_requests()

Created the test under test_bulk_import_request instead of test_project

Also changed some of the code in test_delete() to use the new method

### CODE REVIEW

File: labelbox/schema/project.py [hunk 1]
```diff
@@ -582,6 +582,25 @@ def enable_model_assisted_labeling(self, toggle: bool = True) -> bool:
         return res["project"]["showPredictionsToLabelers"][
             "showingPredictionsToLabelers"]
 
+    def bulk_import_requests(self):
+        """ passpasspass
```
Comment (msokoloff1) [inline labelbox/schema/project.py:586]: prob need to update this lol

File: labelbox/schema/project.py [hunk 2]
```diff
@@ -582,6 +582,25 @@ def enable_model_assisted_labeling(self, toggle: bool = True) -> bool:
...
+        """
```
Comment (msokoloff1) [inline labelbox/schema/project.py:587]: Also does this endpoint return the most recent imports first? If so lets add that to the docstring.

File: labelbox/schema/project.py (hunk 1 above)
Comment (jtsodapop) [reply labelbox/schema/project.py]: Updated! Also goes by oldest first

**Review by msokoloff1 — APPROVED** [review]
Nice!

**Pull request merged**